dts/arm/ene_kb1200/ene_kb1200-pinctrl.dtsi

```diff
@@ -0,0 +1,10 @@
+/*
+ * Copyright (c) 2023 ENE Technology Inc.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr/dt-bindings/pinctrl/ene-kb1200-pinctrl.h>
+
+&pinctrl {
+};
```

Comment:
can you explain why is this file needed?

Reply:
I think that add a empty frame then add nodes for uart device.